ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -752,4 +752,14 @@ MAU_US_SBO_1000_1:
   - MAU_SFSS_SFVSC_HTSC_BPC_SHM
   - DEPRECATED
   uses:
-  - humidification_run_command
\ No newline at end of file
+  - humidification_run_command
+  
+MAU_DFMSC_DFVSC_FDPM_DTC:
+  description: "Makeup Air Unit with discharge air monitoring along with fan speed control"
+  implements:
+  - MAU
+  - DFMSC
+  - DFVSC
+  - FDPM
+  - DTC
```

Comment:
How does it do this? CHW or DX? Please replace with the appropriate control.

Reply:
We are monitoring the MAU and have changed the abstract from DTC to DTM in the above canonical.